### PR TITLE
Update README-ZH.md

### PR DESCRIPTION
**Pull request by Saafo**
Add Feature Requests and Known Bugs modules.

### CODE REVIEW

File: README-ZH.md [hunk 1]
```diff
@@ -183,3 +183,37 @@ npm install --global mocha
...
+
+## 功能请求
+
+在你打开一个新Issue之前，请先检查你的需求是否属于以下类别：
+
+|名称|优先级|需要帮助|
+|---|---|---|
+|[语言规范](https://github.com/LingDong-/wenyan-lang/issues/1)|  ***** |  |
+|[类](https://github.com/LingDong-/wenyan-lang/issues/31) / [对象文法](https://github.com/LingDong-/wenyan-lang/issues/20) |  *** | |
+|[import导入语句](https://github.com/LingDong-/wenyan-lang/issues/100) |  *** | |
```
Comment (Saafo) [inline README-ZH.md]: I'm not sure whether "import导入语句" or "import语句" is better.

File: README-ZH.md [hunk 2]
```diff
@@ -183,3 +183,37 @@ npm install --global mocha
...
+|[函数式程序设计](https://github.com/LingDong-/wenyan-lang/issues/99) |  *** | |
+|更严格的编译器 |  **** | |
+|其他语言的编译器 |  ** | √ |
+|[编辑器的插件](https://github.com/LingDong-/wenyan-lang/issues/77) |  ** | √ |
+|将 [js](https://github.com/LingDong-/wenyan-lang/issues/47)/[py](https://github.com/LingDong-/wenyan-lang/issues/67)/[anything](https://github.com/LingDong-/wenyan-lang/issues/73) 转换回 wenyan(文言) |  * | √ |
+|[转义](https://github.com/LingDong-/wenyan-lang/issues/84)/[生成](https://github.com/LingDong-/wenyan-lang/issues/128) 特殊符号 |  *** | |
+|[对「「」」的替换语法](https://github.com/LingDong-/wenyan-lang/issues/81)|  ** | |
+|[对 。的替换语法](https://github.com/LingDong-/wenyan-lang/issues/93)|  ** | |
+|在线 IDE 的 [字体](https://github.com/LingDong-/wenyan-lang/issues/5) 和 [垂直文本](https://github.com/LingDong-/wenyan-lang/issues/9) |  ** | |
+|[将注释呈现为小型内联文本](https://github.com/LingDong-/wenyan-lang/issues/148) | ** | |
```
Comment (Saafo) [inline README-ZH.md]: also not sure "内联" or "行内" is better. If you think "行内" is better, feel free to modify that!

File: README-ZH.md (hunk 1 above)
Comment (Saafo) [reply README-ZH.md]: I recommend "import语句" now. 🤣You can modify it.

**Comment by LingDong-** [comment]
Thanks a lot for the translation!

**Pull request merged**